api/pkg/utils/utils.go

```diff
@@ -17,13 +17,17 @@ package utils
 import (
 	"encoding/json"
 	"errors"
+	"math"
 	"math/rand"
 	"os"
 	"reflect"
+	"strconv"
 
 	log "github.com/sirupsen/logrus"
 )
 
+var DEFAULT_TIMEOUT_SEC int64 = 5 // Same as the default timeout of go-micro.
```

Comment:
How about making this as a const, as we dont expect runtime change of this value.
const DEFAULT_TIMEOUT_SEC [int64 ] = 5

Reply:
Thanks, done.